doc/CLI-user-manual/README.md

```diff
@@ -5,109 +5,86 @@ sidebar_label: User Manual
-
-> [!NOTE]
->Unless you edit your shell's `$PATH` variable, you need to invoke kc with a `./` prefix to run the script in the current directory:
-
-```sh
-$ ./kc
-```
-
-Begin by creating a new identity. This will be described in more detail later, but try it now with your own first name:
-
-```sh
-$ kc create-id yourName
-did:mdip:test:z3v8AuaYd1CGfC6PCQDXKyKkbt5kJ4o3h2ABBNPGyGNQfEQ99Ce
-```
-
-The long string returned starting with `did` will be unique to you. This is your new Decentralized IDentity (DID for short).
-
-Think of a DID as a secure reference. Only the owner of the reference can change what it points to. What makes it decentralized is that anyone can discover what it points to without involving a third party.
-
-Creating a new ID automatically creates a new wallet for your ID, which we will describe next.
\ No newline at end of file
+## Install Keychain & MDIP
+
+The Keychain project is provided as a set individual processes that make up the components of an MDIP node. These components can optionally be built into Docker containers managed with docker compose, which is what this page will cover.
+
+This process will clone the repository and run the scripts to build and run the Docker containers.
+
+#### Prerequisites
+
+Before following these steps, ensure that the system in question has:
+
+- `git`,
+- `docker` with the Docker Compose plugin,
+- `node` version >=18.
+
+1. Clone the git repository:
+
+    ```sh
+    git clone https://github.com/keychainMDIP/kc
+    cd kc
+    ```
+
+1. Install the project dependencies:
+
+    ```sh
+    npm i
+    ```
+
+1. Create a `.env` file to define the following environment variables:
+
+    ```sh title=".env"
+    KC_GATEKEEPER_URL=http://localhost #Used by the kc CLI to communicate with the gatekeeper.
+    KC_GATEKEEPER_PORT=4224 #If you change this port, update this variable on the system running kc (if separate).
+    KC_NODE_NAME= #The name by which other nodes will identify yours. If blank, your node will show as "anon".
+    KC_NODE_ID= #We will get this value in the next steps.
+    ```
+
+1. Before running a full node with all mediators, we'll first invoke the `docker-compose.yml` file. This configuration only creates a basic node:
+
+    ```sh
+    docker compose up
+    ```
+
+    This may take a few minutes while it builds and deploys the docker images, and syncs network data. When the `hyperswarm-1` and `gatekeeper-1` containers begin logging, you can proceed.
+
+1. In a separate terminal, create an ID for your node. Replace `<NODE-NAME>` with a preferred name to identify this ID in your wallet:
+
+    ```sh
+    ./kc create-id <NODE-NAME>
+    ```
+
+    Copy the output (`did:mdip:...`) to use as the value of `KC_NODE_ID` in `.env`.
+
+1. In the terminal running `docker compose`, execute `CTRL+C` to stop the process.
+
+1. Now you can run your node with additional containers. The `start-node.sh` script will rebuild and start your containers with the bitcoin mediator included:
```

Comment:
Depends on #138

Reply:
start-node with no parameters will start the node with the hyperswarm registry only
start-node -r TESS will start with hyperswarm + Tesseract blockchain registries
start-node -r BTC will start with hyperswarm + Tesseract + Bitcoin blockchain registries

adding a -d will start the node in the background

stop-node takes the same -r parameters as start-node but it is optional since start-node caches the parameter in last-compose-file.txt